tetris.py

```diff
@@ -489,5 +467,28 @@ def timer(self):
             self.topbar_time.config(text=f"{int(game_time / 60):02d}:{int(game_time % 60):02d}")
             self.topbar_time.after(1000, self.timer)
 
+    def new_game(self, event=None):
```

Comment:
This never gets an event, because it's no longer used with bind and buttons don't give an event to their functions:
```suggestion
    def new_game(self):
```

Reply:
Removed